importer/make_KMB_info.py

```diff
@@ -584,6 +587,16 @@ def __init__(self, initial_data, kmb_info):
         self.log = kmb_info.log
         self.commons = pywikibot.Site('commons', 'commons')
 
+    def get_exact_match_church(self):
+        """Try to find correct category for church in Sweden."""
+        if self.kommun:
+            muni_cat_name = self.kmb_info.mappings['kommun'][self.kommun]['commonscat']
+            churches_municip = self.kmb_info.mappings["churches"].get(muni_cat_name)
+            if churches_municip and self.namn in churches_municip:
+                exact_category_title = churches_municip[self.namn][9:]
```

Comment:
I suggest we change load_church_cats to omit the "Category:" prefix instead.

Reply:
Done.